src/Tasks/LinkableMigrationTask.php

```diff
@@ -0,0 +1,91 @@
+<?php declare(strict_types=1);
+
+namespace SilverStripe\LinkField\Tasks;
+
+use SilverStripe\Dev\BuildTask;
+use SilverStripe\LinkField\Models\EmailLink;
+use SilverStripe\LinkField\Models\ExternalLink;
+use SilverStripe\LinkField\Models\FileLink;
+use SilverStripe\LinkField\Models\PhoneLink;
+use SilverStripe\LinkField\Models\SiteTreeLink;
+
+/**
+ * @deprecated 4.0.0 Will be removed without equivalent functionality.
+ */
+class LinkableMigrationTask extends BuildTask
+{
+    use MigrationTaskTrait;
+    use ModuleMigrationTaskTrait;
+
+    private static $segment = 'linkable-migration-task';
+
+    protected $title = 'Linkable to Linkfield Migration Task';
+
+    protected $description = 'Migrate from sheadawson/silverstripe-linkable to silverstripe/linkfield';
+
+    /**
+     * Enable via YAML configuration if you need to run this task
+     */
+    private static ?bool $is_enabled = true;
```

Comment:
```suggestion
    private static ?bool $is_enabled = false;
```
Must be disabled by default

Reply:
Updated